sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/SyncSample.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.digitaltwins.core.models.DigitalTwinsGetByIdResponse;
+import com.azure.identity.ClientSecretCredentialBuilder;
+
+public class SyncSample
+{
+    public static void main(String[] args)
+    {
+        String tenantId = System.getenv("TENANT_ID");
+        String clientId = System.getenv("CLIENT_ID");
+        String clientSecret = System.getenv("CLIENT_SECRET");
+        String endpoint = System.getenv("DIGITAL_TWINS_ENDPOINT");
+        String digitalTwinId = System.getenv("DIGITAL_TWIN_ID");
```

Comment:
If you are interested in trying it out, these values can be copied from your C# ADT SDK config